src/main/histFileMod.F90

```diff
@@ -402,6 +412,7 @@ subroutine hist_printflds()
     ! the CTSM's web-based documentation.
 
     ! First sort the list to be in alphabetical order
+    ! TODO Is t = 1 argument needed?
     call sort_hist_list(1, nallhistflds, allhistfldlist)
```

Comment:
```suggestion
    call sort_hist_list(nallhistflds, allhistfldlist)
```
As far as I can tell, the t = 1 argument and later the plain t argument serve no purpose in subroutine sort_hist_list.